experimental/java-spring-boot2-liberty/stack.yaml

```diff
@@ -0,0 +1,13 @@
+id: java-spring-boot2-liberty
+name: Java Spring Boot2 application built using Maven on OpenLiberty
+version: 0.1.0
+description:  Sample Java Spring Boot2 application built using Maven on OpenLiberty
+icon: https://
+keywords:
+- Java
+- Spring Boot 2
+- Maven
+- Open Liberty
+maintainer: Ozzy <ozzy@ca.ibm.com>
+default-template: default
+license: Apache-2.0
```

Comment:
Could you please a language field to the `stack.yaml`. It's now required which is why the travis build failed. In this case it would be `language: java`

Reply:
Done, also updated the rest of the stack with other changes from current microprofile stack (where applicable for spring rather than microprofile)